frontend/packages/topology/src/utils/useHover.ts

```diff
@@ -6,6 +6,7 @@ const useHover = <T extends Element>(
 ): [boolean, (node: T) => (() => void) | undefined] => {
   const [hover, setHover] = React.useState<boolean>(false);
   const unmountRef = React.useRef(false);
+  const timeoutDelay = delay !== undefined ? delay : 200;
```

Comment:
Use a default value in the param
```
delay? = 200,
```

I'd like to see both a separate `delayIn` and `delayOut` but we can add as needed in future.

Reply:
I've added them